unix/writev_stubs.c

```diff
@@ -0,0 +1,69 @@
+#include <caml/mlvalues.h>
```

Comment:
```suggestion
#define _XOPEN_SOURCE /* IOV_MAX */
#include <caml/mlvalues.h>
```
Linux doesn't export IOV_MAX unless you define _XOPEN_SOURCE or _GNU_SOURCE